pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -31,15 +31,31 @@ import (
 
 	"istio.io/istio/pilot/pkg/features"
 	"istio.io/istio/pilot/pkg/serviceregistry/kube/controller"
+	kubelib "istio.io/istio/pkg/kube"
 	"istio.io/istio/pkg/spiffe"
 	"istio.io/istio/security/pkg/cmd"
 	"istio.io/istio/security/pkg/pki/ca"
+	"istio.io/istio/security/pkg/pki/ra"
 	caserver "istio.io/istio/security/pkg/server/ca"
 	"istio.io/istio/security/pkg/server/ca/authenticate"
 	"istio.io/pkg/env"
 	"istio.io/pkg/log"
 )
 
+type caOptions struct {
+	CAType       string
```

Comment:
Please add comment about what are the values CAType can be.

Reply:
Will do. This will inherit from the MeshConfiguration so will add comments to the MeshConfig API as well